packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -9,6 +9,8 @@ import { IOverflowSet, IOverflowSetItemProps, IOverflowSetProps, IOverflowSetSty
 
 const getClassNames = classNamesFunction<IOverflowSetStyleProps, IOverflowSetStyles>();
 
+const defaultIsReversed = false;
```

Comment:
I think you can just keep this inline with your destructure

Reply:
I have to destructure twice though, so I'd rather it be tied to a constant.